integration-tests/gradle/src/test/resources/conditional-dependencies/ext-quarkus-b/deployment/src/main/java/org.acme.quarkus.ext.a.deployment/AcmeQuarkusExtProcessor.java

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
I've noticed the packages aren't laid out as directories on the filesystem, i.e. `src/main/java/org.acme.quarkus.ext.a.deployment/AcmeQuarkusExtProcessor.java ` was that on purpose?

Reply:
Oh no, it's not, I will update that too.